lib/lsperfm/core/stats.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+require "thread"
+
+Thread.abort_on_exception = true
+
+module LogStash::PerfM
```

Comment:
Yes, test is completely misleading here in terms of description. What about calling this ```PerformanceMeter``` is long, but it fits the requirement of being explicit.